impeller/aiks/aiks_playground_inspector.h

```diff
@@ -21,6 +21,17 @@ class AiksInspector {
       AiksContext& aiks_context,
       const std::function<std::optional<Picture>()>& picture_callback);
 
+  // Resets (releases) the underlying |Picture| object.
+  //
+  // Underlying issue: <https://github.com/flutter/flutter/issues/134678>.
+  //
+  // The tear-down code is not running in the right order; it's torn down after
+  // we shut down the |ContextVK|, which means that the Vulkan allocator still
```

Comment:
Its not that VMA still has a reference, its that when shutting down VMA we have retained dedicated allocations without freeing them

Reply:
Done.